sci-biology/hmmer/hmmer-3.1_beta2.ebuild

```diff
@@ -2,14 +2,16 @@
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=5
+EAPI=6
 
 inherit eutils
 
-MY_PV="3.1b2"
+
```

Comment:
Running `repoman full` returns:
```
RepoMan scours the neighborhood...
  ebuild.minorsyn               1
   sci-biology/hmmer/hmmer-3.1_beta2.ebuild: Useless blank line on line: 9
```

For future PRs, please use `repoman commit`, which is automatically checking your ebuild (running `repoman full`), updates the Manifest and so on...